### PR TITLE
Turbo Drive lesson: Fix punctuation, capitalizations, break up long sentence

### PR DESCRIPTION
**Pull request by Overlandcoder**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

### CODE REVIEW

File: ruby_on_rails/assets_and_navigation/turbo_drive.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### Introduction
 
-Rails has always promoted itself as a framework that makes building a web application easy, but in a world built around the demands of users expecting lightning fast responses as well as increasingly complex UI it wasn't easy for Rails to offer a compelling vision for how they saw a way through that complexity while sticking to core Rails philosophies. However just before the release of Rails 7 we finally got a glimpse of how the Rails team saw the future of meeting these demands. Hotwire!
+Rails has always promoted itself as a framework that makes building a web application easy. But, in a world built around the demands of users expecting lightning fast responses as well as increasingly complex UI, it wasn't easy for Rails to offer a compelling vision for how they saw a way through that complexity while sticking to core Rails philosophies. However, just before the release of Rails 7 we finally got a glimpse of how the Rails team saw the future of meeting these demands. Hotwire!
```
Comment (linkonsat) [inline ruby_on_rails/assets_and_navigation/turbo_drive.md:3]: The but needs to be capitalized, but typically we wouldn't put a comma after the "But". That's my only nitpick and then I would be happy to approve this PR after the comma is removed.

**Review by linkonsat — CHANGES_REQUESTED** [review]
Just one small nitpick and then I would be happy  to approve this PR. Thanks for doing this as well!

File: ruby_on_rails/assets_and_navigation/turbo_drive.md (hunk 1 above)
Comment (Overlandcoder) [reply ruby_on_rails/assets_and_navigation/turbo_drive.md]: You're right, that would be better. I made the change now.

**Review by linkonsat — APPROVED** [review]

**Comment by linkonsat** [comment]
Awesome. Thanks for making these changes!

**Pull request merged**